include/aspect/stokes_matrix_free.h

```diff
@@ -443,12 +449,6 @@ namespace aspect
        */
       void build_preconditioner() override;
 
-      /**
-       * Get the workload imbalance of the distribution
-       * of the level hierarchy.
-       */
-      double get_workload_imbalance();
-
       /**
```

Comment:
After requiring dealii-9.2, this function is unnecessary (`MGTools::workload_imbalance()` does the same thing). I imagine I am the only person using this function, but if I need to deprecate instead of deleting, I am happy to.

Reply:
it is unused in ASPECT? Then we can delete it.

Reply:
yes, unused